.github/workflows/test-build-deploy.yml

```diff
@@ -42,6 +42,15 @@ jobs:
           bundle exec rake db:create
           bundle exec rake db:schema:load
 
+      - name: Brakeman
+        uses: reviewdog/action-brakeman@v2
+        with:
+          brakeman_version: gemfile
+          # --force required as no /app dir as expected
+          # https://github.com/presidentbeef/brakeman/issues/67#issuecomment-4947358
+          brakeman_flags: --force
+          reporter: github-pr-review
```

Comment:
What made you choose github-pr-review over the default (github-pr-check)?

Reply:
Its so the reviewdog error results display in the "conversation" history rather than the "checks" section.

https://github.com/marketplace/actions/run-brakeman-with-reviewdog#with-github-pr-review

Reply:
It sounds like the error gets added as a comment where the issue occurs with github-pr-check. If there are several issues maybe that would be easier to understand where they occur?

Reply:
Sure thing I'll remove to keep the default